components/index/CardsMonitoring/PositiveRate/Card.vue

```diff
@@ -133,13 +133,16 @@ export default {
       }
     }
 
+    const units = [this.$t('ポイント'), '%']
```

Comment:
```suggestion
    const units = [this.$t('ポイント') as string, '%']
```

このようにしなくても大丈夫な感じですかね...？

Reply:
@kaizumaki 
レビューありがとうございます！ただ、当該ファイルはtsで書かれているものではないので、その様に記載していないと言う意図になります。

Reply:
わ、ほんとですね！ではそのままでOKです！
（いつかts化したい...）

Reply:
ですね…
TSにするためだけのissueがあったら良さそうです！

Reply:
そうですね！一連のサイトリニューアルのリリースが終わったらissue立てますね。